.gitignore

```diff
@@ -11,3 +11,4 @@ benchmark/results
 .prettierrc.js
 .prettierrc
 logs/
+.history/
```

Comment:
Why is it necessary?

Reply:
I have history plugin so it does not hurt to include this